cmd/proxy/proxy.go

```diff
@@ -356,22 +313,16 @@ func (c *Command) RoundTrip(req *http.Request) (*http.Response, error) {
 		return resp, err
 	}
 
-	if len(c.rules) > 0 {
-		begin := time.Now()
-		action := new(action)
-		for _, rule := range c.rules {
-			if action.skip {
-				break
-			}
-			if rule.match(req, resp) {
-				action = action.merge(rule.apply(req, resp))
-			}
-		}
+	begin := time.Now()
+	shouldTrace, err2 := shouldTrace(req, resp, c.config)
```

Comment:
`err` is nil at this point, so just reuse it here instead of a `err2`. we're doing `return resp, nil` at the end, so you can do that in the `if !shouldTrace { ... }` too.